#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_makefile.dpatch by Denis V. Sirotkin <fechiny@gmail.com>
## Original changes was done by Tom Huckstep <tom-debian-polipo@jaguarpaw.co.uk>
##
## DP: Change description

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
	echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
	exit 1;;
esac

exit 0

@DPATCH@

--- polipo-1.0.1.orig/Makefile
+++ polipo-1.0.1/Makefile
@@ -1,4 +1,5 @@
-PREFIX = /usr/local
+PREFIX = /usr
+TARGET = $(DESTDIR)
 BINDIR = $(PREFIX)/bin
 MANDIR = $(PREFIX)/man
 INFODIR = $(PREFIX)/info
@@ -62,7 +63,7 @@
 
 DEFINES = $(FILE_DEFINES) $(PLATFORM_DEFINES)
 
-CFLAGS = $(MD5INCLUDES) $(CDEBUGFLAGS) $(DEFINES) $(EXTRA_DEFINES)
+override CFLAGS += $(MD5INCLUDES) $(CDEBUGFLAGS) $(DEFINES) $(EXTRA_DEFINES)
 
 SRCS = util.c event.c io.c chunk.c atom.c object.c log.c diskcache.c main.c \
        config.c local.c http.c client.c server.c auth.c tunnel.c \
@@ -91,10 +92,13 @@
 	mkdir -p $(TARGET)$(BINDIR)
 	mkdir -p $(TARGET)$(LOCAL_ROOT)
 	mkdir -p $(TARGET)$(LOCAL_ROOT)/doc
+	mkdir -p $(TARGET)/etc/polipo
 	rm -f $(TARGET)$(BINDIR)/polipo
 	cp -f polipo $(TARGET)$(BINDIR)/
 	cp -f html/* $(TARGET)$(LOCAL_ROOT)/doc
 	cp -f localindex.html $(TARGET)$(LOCAL_ROOT)/index.html
+	cp -f config.sample $(TARGET)/etc/polipo/config
+	cp -f forbidden.sample $(TARGET)/etc/polipo/forbidden
 
 install.man: all
 	mkdir -p $(TARGET)$(MANDIR)/man1
